_replication/scheduling/advanced-cron-scheduling.md

```diff
@@ -0,0 +1,334 @@
+            <td rowspan="{{ total-examples | strip }}">
+                {{ explanation.explanation | markdownify }}
+            </td>
+
+            {% for example in explanation.examples %}
+                {% case forloop.first %}
+                    {% when true %}
+                            <td>
+                                {{ example.expression | markdownify | strip }}
+                            </td>
+                            <td>
+                                {{ example.translation | strip }}
+                            </td>
+                        </tr>
+                    {% else %}
+                        <tr>
+                            <td>
+                                {{ example.expression | markdownify | strip }}
+                            </td>
+                            <td>
+                                {{ example.translation | strip }}
+                            </td>
+                        </tr>
+                {% endcase %}
+            {% endfor %}
+    {% endfor %}
+</table>
+
+
+---
+
+## Example replication schedules {#examples}
+
+{% capture resource-callout %}
+**Want to test an expression?** Use [{{ cron.resource-names.crontab }}]({{ cron.resource-urls.crontab }}){:target="new"} to test expressions and see the schedule that will be created.
+{% endcapture %}
+
+{% include tip.html content=resource-callout %}
+
+Based on scheduling feedback we've collected, we've put together a list of some of the most commonly requested replication schedules. The cron expressions in the table below can be copied and pasted directly into an integration's settings page in Stitch.
+
+**Note**: When creating your own cron expressions, keep in mind that there are some [limitations](#limitations).
+
+<table class="attribute-list">
+    <tr>
+        <td>
+            <strong>
+                Expression
+            </strong>
+        </td>
+        <td>
+            <strong>
+                Translation
+            </strong>
+        </td>
+    </tr>
+    {% for example in cron.examples %}
+        <tr>
+            <td>
+                {{ example.expression | markdownify }}
+            </td>
+            <td>
+                {{ example.translation }}
+            </td>
+        </tr>
+    {% endfor %}
+</table>
+
+---
+
+## Limitations of cron scheduling {#limitations}
+
+While using cron expressions will give you the most control over your integrations' replication schedules, there are some limitations to keep in mind:
+
+1. **Advanced scheduling isn't available for all integrations**. Some database integrations don't currently support advanced scheduling. We are working on converting these integrations into Singer-powered taps, at which point advanced scheduling will be available.
+
+2. **Days of the week in specific months can't be specified** due to the implementation of cron Stitch uses. POSIX (the name of the cron library) [has a unique method for specifying days](#specifying-days).
```

Comment:
posix is not a library but a standard http://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html

Reply:
Fine, fine ;)